CONTRIBUTING.md

```diff
@@ -46,14 +46,38 @@ $ bundle exec rake console
 
 Tests are very important part of google-cloud-ruby. All contributions should include tests that ensure the contributed code behaves as expected.
 
+To run the unit tests, documentation tests, and code style checks together:
+
+``` sh
+$ rake ci
```

Comment:
Is it worth noting that users can run `rake ci:acceptance` (with handy alias of `rake ci:a`) to run all previous tasks plus acceptance tests?